src/main/java/duke/command/FindCommand.java

```diff
@@ -0,0 +1,43 @@
+package duke.command;
+
+import duke.DukeException;
+import duke.TaskList;
+import duke.tasks.Task;
+import duke.tasks.taskType;
+import duke.ui.Ui;
+
+import java.util.ArrayList;
+
+public class FindCommand extends Command{
```

Comment:
I think you might've missed the space after Command too, can add them to follow coding standard :)